pcf-infrastructure/advanced-certificate-rotation.html.md.erb

```diff
@@ -5,8 +5,9 @@ owner: CredHub Maestro
 
 This topic describes how to rotate certificates and Certificate Authorities (CAs) in CredHub using CredHub Maestro.
 
-This topic only covers rotating CredHub-managed certificates. To rotate both Ops Manager- and CredHub-managed certificates, see [Rotating Certificates](api-cert-rotation.html).
+This topic only covers rotating CredHub-managed certificates. To rotate Ops Manager managed certificates, see [Rotating Certificates](api-cert-rotation.html).
 
+<p class="note warning"><strong>Warning:</strong> Not every tile currently supports certificate rotation so there is a potential for downtime.</p>
```

Comment:
Hey @vullob! just following up on this to make sure we are clear on how to avoid this impact. Is there a way for operators to tell whether their tile support certificate rotation? Is there something they can check before using Maestro?

Reply:
ok, chatted with tom kennedy and marty spiewak -- we discussed publishing a list of which specific tiles support this so customers know what is safe/unsafe. we'll update this note with a link when available.